screens/ConfirmationScreen.js

```diff
@@ -34,6 +35,9 @@ export default class ConfirmationScreen extends React.Component {
     if (isLoading) {
       return null;
     }
+
+    // TODO make this a value in AirTable
+    const rewardValue = 5;
     return (
       <ColumnContainer style={{ width: '100%', height: '100%', justifyContent: 'center', alignItems: 'center' }}>
         <ColumnContainer style={{ width: '33%', paddingBottom: 48 }}>
```

Comment:
Sorry unrelated to this PR but can you remove Purchase Recorded (based on latest design updates)

Reply:
![image](https://user-images.githubusercontent.com/21094576/77259672-68317580-6c40-11ea-89ae-7774094ae114.png)
Designs weren't exactly up-to-date spacing-wise, so I just had it look like this for now